### PR TITLE
Update market-hours-database.json

### LINKED ISSUE #3599: Add Christmas and New Year Eve to OANDA Forex MHDB
#### Expected Behavior

Market hours database should replicate actual holidays 

#### Actual Behavior

Basic holidays missing for 2018-2019.

#### Potential Solution

Add missing holidays.

#### Reproducing the Problem

N/A

#### System Information

N/A

#### Checklist

- [ ] I have completely filled out this template
- [ ] I have confirmed that this issue exists on the current `master` branch
- [ ] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)

- [ ] I have provided detailed steps to reproduce the issue

### PR DESCRIPTION
**Pull request by Jay-Jay-D**
#### Description

Add New Year and Chrismas holidays to Forex Oanda 

#### Related Issue

Closes #3599 

#### Motivation and Context

Data for those days seem missing data but actually, those dates are holidays.

#### Requires Documentation Change

N/A

#### How Has This Been Tested?

Local test.

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [ ] I have added tests to cover my changes. 
- [x] All new and existing tests passed.
- [x] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: Data/market-hours/market-hours-database.json [hunk 1]
```diff
@@ -23329,7 +23329,40 @@
         }
       ],
       "saturday": [],
-      "holidays": [],
+      "holidays": [
```
Comment (mchandschuh) [inline Data/market-hours/market-hours-database.json]: do these exchanges not have data before 2008?

Comment (StefanoRaggi) [reply Data/market-hours/market-hours-database.json]: Seems we have Oanda Forex data going back until 2002, although for the first 3 years we have only daily data points

File: Data/market-hours/market-hours-database.json [hunk 2]
```diff
@@ -23329,9 +23329,47 @@
...
+        "12/25/2018",
+        "12/25/2019",
+        "12/25/2020",
+        "12/25/2023",
+        "12/25/2024",
+        "12/25/2025"
+      ],
       "earlyCloses": {
-        "12/31/2018": "17:00:00"
+        "12/31/2018": "21:00:00",
```
Comment (StefanoRaggi) [inline Data/market-hours/market-hours-database.json]: Why was the time changed from "17:00:00" to "21:00:00"?
Early close time comparisons are done using the exchange time zone:
https://github.com/QuantConnect/Lean/blob/master/Common/Securities/SecurityExchangeHours.cs#L336

Comment (StefanoRaggi) [reply Data/market-hours/market-hours-database.json]: Looks like early closes are also missing for `"12/24/20xx": "17:00:00"`

**Pull request merged**